tpl/transform/transform_test.go

```diff
@@ -168,6 +168,33 @@ func TestMarkdownify(t *testing.T) {
 	}
 }
 
+func TestAsciidocify(t *testing.T) {
+	t.Parallel()
+
+	ns := New(newDeps(viper.New()))
+
+	for i, test := range []struct {
+		s      interface{}
+		expect interface{}
+	}{
+		{"Hello *World!*", template.HTML("Hello <strong>World!</strong>")},
+		{[]byte("Hello **Bytes World!**"), template.HTML("Hello <strong>Bytes World!</strong>")},
+		{tstNoStringer{}, false},
+	} {
+		errMsg := fmt.Sprintf("[%d] %s", i, test.s)
+
+		result, err := ns.Markdownify(test.s)
```

Comment:
Your test is using Markdownify instead of Asciidocify. I guess this should be adjusted to validate actual AsciiDoc code instead.